tests/runner.py

```diff
@@ -663,9 +664,13 @@ def do_run(self, src, expected_output, args=[], output_nicerizer=None,
     for engine in js_engines:
       #print 'test in', engine
       js_output = self.run_generated_code(engine, filename + '.o.js', args, output_nicerizer=output_nicerizer, assert_returncode=assert_returncode)
+      js_output = js_output.replace('\r\n', '\n')
```

Comment:
We should have many multiline tests before, is there something special that required changing this flow?